utils/docker/Dockerfile.leap.15

```diff
@@ -30,9 +30,10 @@ RUN zypper --non-interactive --no-gpg-checks refresh; \
            python-devel python3-devel valgrind-devel hwloc-devel      \
            openmpi3-devel Modules man
 
-RUN pip install --upgrade pip; \
+RUN update-ca-certificates; \
+    pip install --upgrade pip; \
     pip install -U setuptools; \
-    pip install -U wheel;      \
+    pip install -U wheel; \
```

Comment:
We are lining up the \ under each other to easily determine blocks of continuous code.  The whitespace you removed here was intentional.

Reply:
Okay. Thanks for letting me know.